src/material/select/select.ts

```diff
@@ -506,13 +511,13 @@ export abstract class _MatSelectBase<C> extends _MatSelectMixinBase implements A
     // Force setter to be called in case id was not specified.
     this.id = this.id;
 
-    if (defaults) {
-      if (defaults.disableOptionCentering != null) {
-        this.disableOptionCentering = defaults.disableOptionCentering;
+    if (_defaultOptions) {
+      if (_defaultOptions.disableOptionCentering != null) {
+        this.disableOptionCentering = _defaultOptions.disableOptionCentering;
```

Comment:
I think we can simplify these to something like `this.disableOptionCentering = _defaultOptions.disableOptionCentering ?? false;`.